viper/parser/stmt.py

```diff
@@ -0,0 +1,269 @@
+import ast
+
+from viper.exceptions import (
+    ConstancyViolationException,
+    StructureException,
+    TypeMismatchException,
+    VariableDeclarationException,
+)
+from viper.functions import (
+    stmt_dispatch_table,
+)
+from .parser_utils import LLLnode
+from .parser_utils import (
+    getpos,
+    make_byte_array_copier,
+    base_type_conversion,
+    unwrap_location
+)
+from viper.types import (
+    BaseType,
+    ByteArrayType,
+    ListType,
+)
+from viper.types import (
+    get_size_of_type,
+    is_base_type,
+    parse_type,
+)
+from viper.types import (
+    are_units_compatible,
+    set_default_units,
+)
+
+
+class Stmt(object):
+    def __init__(self, stmt, context):
+        self.stmt = stmt
+        self.context = context
+        self.stmt_table = {
+            ast.Expr: self.expr,
+            ast.Pass: self.parse_pass,
+            ast.AnnAssign: self.ann_assign,
+            ast.Assign: self.assign,
+            ast.If: self.parse_if,
+            ast.Call: self.call,
+            ast.Assert: self.parse_assert,
+            ast.For: self.parse_for,
+            ast.AugAssign: self.aug_assign,
+            ast.Break: self.parse_break,
+            ast.Return: self.parse_return,
+        }
```

Comment:
Just something small thing I would opt for readability. I like to have something like stmt_table on the class straight: 

```python

class Stmt():
   stmt_table = { ... }

Reply:
I like that a lot better, great suggestion!

Reply:
Sure, makes sense, since it's just another way to do a bunch if/elifs.